### PR TITLE
Update the Doctor's powers

### PR DESCRIPTION
**Pull request by GrayHat4Life**
I'll provide the evidence for these powers tomorrow in-game

### CODE REVIEW

**Comment by GrayHat4Life** [comment]
Alright, I'm giving all the Doctors the same powers (though maybe different items in some cases). Here's the proof of the Doctor's psychic abilities:
The Doctor in a mind wrestling match
https://www.youtube.com/watch?v=51a1hoSn4uY
A Time Lord "mind link"
https://www.youtube.com/watch?v=kb5YZK5lNOk
Telepathic information transfer
https://www.youtube.com/watch?v=3_FcNoOPC-E

All Time Lords are telepathic (the proof of this is a wiki since I can't think of a specific instance where it's directly said in the show/books/comics/movie), so I gave the Doctor basic telepathic powers. If mind control doesn't fall under that category in your mind, feel free to take it away.

**Comment by GrayHat4Life** [comment]
I also have proof of his enhanced physical attributes (e.g., resistance and haste) if you want that as well.

**Comment by GrayHat4Life** [comment]
Here's the proof of resistance (again a Time Lord trait not specific to the Doctor), I think you'll agree that falling from that height should give him at least a little resistance
https://www.youtube.com/watch?v=B0PmYaOBuQk
And here's some further proof of overall physical superiority to humans. Notice near the end of the video how much longer he lasts without oxygen that everyone else. Most of the basic powers I added (run, jump, & haste) stem from this overall physical superiority.
https://www.youtube.com/watch?v=ZqNDkSqL4TA

**Comment by GrayHat4Life** [comment]
If you need more proof of physical powers I'll be happy to oblidge

**Comment by GrayHat4Life** [comment]
It's been established cannon in the show for decades that Time Lords have two hearts, which is why I put two bars (get it?). Here's the first example that comes to my mind:
https://www.youtube.com/watch?v=sDG8GstfLY0
Now for the regen. This is THE most obvious power in my mind. First I'll have to show you a wiki page so you can understand what this is (this isn't my proof, obviously)
http://tardis.wikia.com/wiki/Regeneration
Then some proof (or all of it, really):
https://www.youtube.com/watch?v=uXCpY_3Sac8
This is why there are multiple Doctors, by the way. He keeps regenerating; they're all the exact same person.

File: CommandHelper/LocalPackages/commands/buffs/thefourthdoctor.ms [hunk 1]
```diff
@@ -12,8 +12,17 @@
             'speech': "Would you like a jelly baby?",
             'powers': array(
                 "/buff"
+                "/mindcontrol <playername>"
+                "/stopmindcontrol"
+                "/confuse <playername>"
+                "/mindlock <playername>"
+                "/msg <playername> [message]"
+                "You have access to all warps with your TARDIS"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/thefourthdoctor.ms]: where all the commas bro?

Comment (GrayHat4Life) [inline CommandHelper/LocalPackages/commands/buffs/thefourthdoctor.ms]: Whoops. I'll do that now if I can

**Comment by GrayHat4Life** [comment]
Proof that Time Lords aren't weakened that much by regeneration
https://www.youtube.com/watch?v=93nJQj2Z97M
And the Doctor himself:
https://www.youtube.com/watch?v=4F84WapAH7M
He is a bit disoriented sometimes, but by no means is he out of it

**Comment by GrayHat4Life** [comment]
Plus, it really wouldn't make sense to fans of the show who visit the server if the Doctor didn't have regeneration

**Comment by GrayHat4Life** [comment]
I started a new pull request with the commas fixed this time

**Pull request merged**